src/main/java/roomescape/controller/AdminViewController.java

```diff
@@ -0,0 +1,18 @@
+package roomescape.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class AdminViewController {
+    @GetMapping(value = "/")
+    public String home() {
+        return "home";
+    }
+
+    @GetMapping(value = "/reservation")
+    public String reservation() {
+        return "reservation";
+    }
+
+}
```

Comment:
```suggestion
    @GetMapping(value = "/reservation")
    public String reservation() {
        return "reservation";
    }
}
```
그리고 마지막 메서드에는 개행을 해주지 않습니당